test/units/modules/cloud/docker/test_docker_network.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+
+from ansible.modules.cloud.docker.docker_network import get_ip_version
+
+
+class TestDockerNetwork(unittest.TestCase):
+    """Unit tests for docker_network."""
+
+    def test_get_ip_version(self):
+        """Ensure get_ip_version correctly identifies IP version of a CIDR string."""
+
+        self.assertEqual(get_ip_version('192.168.0.1/16'), 'ipv4')
+        self.assertEqual(get_ip_version('192.168.0.1/24'), 'ipv4')
+        self.assertEqual(get_ip_version('192.168.0.1/32'), 'ipv4')
+
+        self.assertEqual(get_ip_version('fdd1:ac8c:0557:7ce2::/64'), 'ipv6')
+        self.assertEqual(get_ip_version('fdd1:ac8c:0557:7ce2::/128'), 'ipv6')
+
+        with self.assertRaises(ValueError) as e:
```

Comment:
```
self.assertRaises(ValueError, get_ip_version, '192.168.0.1')
```
should work with python 2.6 but you can't check the error message.

Reply:
In the [test container](quay.io/ansible/default-test-container:1.4.1), the backported [unittest2](https://pypi.org/project/unittest2/) is available. So I added a switch to use the backported unittest2 on python < 2.7